config/environments/development.rb

```diff
@@ -45,13 +52,16 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Allow requests for all domains e.g. <app>.dev.gov.uk
-  config.hosts.clear
```

Comment:
This line might still be needed, as it appears to be a GOV.UK specific config.

Reply:
Thanks @brucebolt , I've updated that now 👍